Os/File.cpp

```diff
@@ -0,0 +1,237 @@
+// ======================================================================
+// \title Os/File.cpp
+// \brief common function implementation for Os::File
+// ======================================================================
+#include <Os/File.hpp>
+#include <Fw/Types/Assert.hpp>
+
+#ifdef __cplusplus
```

Comment:
Do we want this ifdef cplusplus block? The file is already including hpp files, so how can it be compiled as a C file?

Reply:
Agreed.  Will commit a fix.